docs/metrics-events.md

```diff
@@ -67,6 +67,8 @@ in a sign-in or sign-up flow:
 |`password.forgot.resend_code.completed`|A password reset email has been re-sent to the user.|
 |`password.forgot.verify_code.start`|A user has clicked on the link in a password reset email.|
 |`password.forgot.verify_code.completed`|A password reset has been successfully completed on the server.|
+|`route.${path}.200`| A route responded with a 200 status code. Example: `route./account/login.200`|
```

Comment:
For people consuming these events in redash, I wonder whether the `/` characters might seem slightly weird. Given that we've settled on `.` as a universal separator of sorts, do you think it is worth us converting these event names to something like `route.account.login.200`?